### PR TITLE
Localisation support

### PR DESCRIPTION
**Pull request by gljk**
I enabled support for l10n localization by updating the code of the extension to work with [The Big E](https://docs.civicrm.org/dev/en/latest/extensions/structure/#the-big-e). With these changes you are able to add different  languages (including Dutch #145 ), by placing an appropriate mo file in the extension directory (e.g. uk.co.compucorp.civicrm.booking/l10n/nl_NL/LC_MESSAGES/booking.mo)

### CODE REVIEW

**Comment by jamienovick** [comment]
@guanhuan do you think we can find some time to review this? Would be good if we can get CiviBooking translatable?

File: CRM/Admin/Form/AdhocChargesItem.php [hunk 1]
```diff
@@ -131,7 +132,7 @@ public function postProcess() {
     // TODO::Make sure we cannot delete if the entity is linked to bookings
     if ($this->_action & CRM_Core_Action::DELETE) {
       CRM_Booking_BAO_AdhocChargesItem::del($this->_id);
-      CRM_Core_Session::setStatus(ts('Selected additional charges item has been deleted.'), ts('Record Deleted'), 'success');
+      CRM_Core_Session::setStatus(ts('Selected additional charges item has been deleted.'), E::ts('Record Deleted'), 'success');
```
Comment (nishant-bhorodia) [inline CRM/Admin/Form/AdhocChargesItem.php]: hey @gljk, 
Thanks for the patch, there are a few ts() which are not converted to E::ts() like on this line.
Is there a specific reason for that or can you please convert those as well?
Regards.

Comment (nishant-bhorodia) [reply CRM/Admin/Form/AdhocChargesItem.php]: ok , please can you fix these as well @gljk .
Thanks.

Comment (gljk) [reply CRM/Admin/Form/AdhocChargesItem.php]: @nishant-bhorodia No problem, I just did that.

**Comment by nishant-bhorodia** [comment]
Thanks @gljk , 
This PR is approved from my side.
FYI @guanhuan 
Regards.

**Pull request merged**